packages/common/src/core/schemas/internal/metrics/MetricSchema.ts

```diff
@@ -155,5 +152,21 @@ export const MetricSchema: IConfigurationSchema = {
         properties: { value: { type: "number" } },
       },
     },
+    "if-static-then-url-format": {
+      if: {
+        type: "object",
+        properties: {
+          type: { const: SOURCE.static },
+        },
+      },
+      then: {
+        type: "object",
+        properties: {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          sourceLink: { format: "url" },
```

Comment:
do we still want the `if: { minLength: 1 }` `then...` logic here?

Reply:
I don't think we need it anymore, since the url formatter lets data with 0 length pass through correctly -- right?